File: cpp_results/instruct_generations/cpp/files/question-16-pass-1.cpp
#include <bits/stdc++.h> 
 using namespace std ;
int alternatingSumOfFirst_N(int N) {
    int sum = 0;
    for (int i = 1; i <= N; i++) {
        sum += (i % 2 == 0)? i : -i;
    }
    return sum;
}

File: results/fine_tuning/test_output_files/question-16-pass-1.py
def alternatingSumOfFirst_N(N):
    if (N == 1):
        return 1
    else:
        sum = 0
        for i in range(1, N + 1):
            if (i % 2 == 0):
                sum = sum + i
            else:
                sum = sum - i
        print(sum)


if __name__ == "__main__":
    N = 5
    alternatingSumOfFirst_N(N)
